### PR TITLE
Fallback to Node.Address if Service.Address is empty when building Consul record

### LINKED ISSUE #156: ConsulServiceImporter should fallback to the Node address if not set on the Service
### Version

Latest

### Context

We changed how our services are being registered with consul and now the `Address` field of the `Service` is no longer populated. 

```json
    "Service": {
      "ID": "service-name",
      "Service": "service-name",
      "Tags": [      ],
      "Address": "",
      "Meta": null,
      "Port": 6565,
      "EnableTagOverride": false,
      "ProxyDestination": "",
      "Connect": {
        "Native": false,
        "Proxy": null
      },
      "CreateIndex": 9064290,
      "ModifyIndex": 9064290
    },
```

### Do you have a reproducer?

No, but the [Consul documentation](https://www.consul.io/docs/discovery/services) states that the Node address should be used if the Service address is empty.

> If no value is specified, the IP address of the agent node is used by default. 

### Extra

https://github.com/hashicorp/consul/issues/4599

### PR DESCRIPTION
**Pull request by jareddellitt**
Motivation:

Resolves #156

### CODE REVIEW

File: vertx-service-discovery-bridge-consul/src/main/java/io/vertx/servicediscovery/consul/ConsulServiceImporter.java
Comment (therockstorm) [inline vertx-service-discovery-bridge-consul/src/main/java/io/vertx/servicediscovery/consul/ConsulServiceImporter.java:249]: Wouldn't we want to fallback if address == null too? The code from core does,

```
def resolveAddress(service: ServiceHealth): String = {
    val addr = service.getService.getAddress
    if (Option(addr).isDefined && addr.nonEmpty) addr
    else service.getNode.getAddress
  }
```

**Review by therockstorm — APPROVED** [review]

**Review by tsegismont — APPROVED** [review]
Thank you @jareddellitt ! Waiting for CI job to complete before merging.

**Pull request merged**
